src/pages/LoginAndRegister/GuestAccountRedirect.vue

```diff
@@ -23,15 +24,18 @@ export default {
 						}
 					}
 				}`,
-			}).then(({ data }) => {
+				}),
+				route,
```

Comment:
Why does `route` need to be passed here to `Promise.all`, can't it just be accessed already inside this method?

Reply:
This is under `prefetch`, we don't have access to `this` method...

Reply:
Right, but `route` is being passed into `preFetch`. I see we are already using `cookieStore` below without adding it to the `Promise` resolution. Does `route` work similarly in this case?

Reply:
Nice call, checking now...

Reply:
Works fine, thanks!